staff/jordi-pujol/e-commerce-protos/scripts/cart.test.js

```diff
@@ -0,0 +1,140 @@
+var cart = new Cart;
+
+
+var socks = new Socks('Calvin Klein', 42, 'black', 9.99);
+var tanga = new Tanga('Wicked Weasel', 32, 'red', 29.95);
+var bra = new Bra('Calvin Klein', 'push-up', 80, 'cream', 39); // model
+var slips = new Slips('Abanderado', 42, 'purple', 13.99);
+
+cart.add(socks)
+cart.add(tanga)
+cart.add(bra)
+cart.add(slips)
+
+suite('cart', function(){
+
+    // .Add()
+
+    describe ('adding elements', function(){
+    
+        it ('should add one element', function(){
+            var socks = new Socks('Calvin Klein', 42, 'black', 9.99);
+            
+            var res = cart.add(socks)
+
+            var expected = 5;
+
+            expect (res === expected,'expected and result should be the same')
+        });
+    })
+
+    describe ('fails adding', function() {
```

Comment:
I would not make a new describe for the fail cases. I would put them in new it but inside the same describe than the succeed ones